README.md

```diff
@@ -1,2 +1,119 @@
 # the-orchard
-A tool for building out required script and link tags for dependencies of a project.
+
+A CLI tool to generate the markup for including dependencies
+
+## Usage
+
+```npm i -DE the-orchard```
+
+Once installed the `orchard` cli tool becomes available. For a good description of all the options and flags, please run:
+
+```orchard --help```
+
+## Updating your html file using the cli
+
+All of your dependencies can be inserted into your html file via the cli!
+
+Putting the following text anywhere in a file will result in that string being replaced with the markup for all of the dependencies in your list.
+
+```html
+<!--THE_ORCHARD-->
+```
+
+We recommend using a separate source file from your output file to allow for source control to ignore the generated file. The CLI command to accomplish this should look something like:
+
+```bash
+orchard -i ./sourceHtmlFile.html -o ./outputHtmlFile.html
+```
+
+### Usage Implications
+
+In order to selectively serve modern code to modern browsers the script tags generated from the orchard utilize the `type="module"` attribute for es6+ builds, and es5 compatible code uses script tags with the `nomodule` and `defer` attributes.
+Browsers that support `type="module"` will only load the tags with that name, and ignore the `nomodule` marked versions.
+Legacy browsers that do not support module will ignore the `nomodule` tag as unknown, but will load the script respecting the `defer` attribute.
+
+The big impact here is that any scripts loaded by the orchard are loaded *asynchronously*, so any other scripts depending on anything loaded here must be loaded with the `defer` attribute in the script tag.
+
+This also has a performance benefit of not blocking the page load time while waiting for the scripts to download and be parsed.
+
+## Registry Data
+
+This package relies on a folder being populated with file defining the dependencies you are concerned with in yaml format. (See Registry Data File Formats below).
+
+The default folder is `orchard`. If you would like an example of a recommended convention for this folder please check out the [demo folder](demo/orchard).
+
+## Registry Data File Formats
+
+The `/registry-data` file formats are documented elsewhere. Follow the link below for more information.
+
+- [dependencies](docs/README.md#dependency-file-format)
+
+## Assumptions when using the The Orchard
```

Comment:
In this section you changed the spelling of "The Orchard".
Also looks like you got two places at least that contain a copy and paste error.

```suggestion
## Assumptions when using The Orchard
```